concourse/tasks/render-templates.yaml

```diff
@@ -1,13 +1,7 @@
 platform: linux
 
-resource_types:
-- name: guest-image-private
-  type: registry-image
-  source:
-    repository: gcr.io/compute-image-tools/registry-image-forked
-
 image_resource:
-  type: registry-image-private
```

Comment:
Can we do the other way around and stick to `registry-image-private`? Meaning changing the resource type name.